splink/comparison_level_library.py

```diff
@@ -225,3 +226,75 @@ def columns_reversed_level(
         level_dict["tf_adjustment_column"] = tf_adjustment_column
 
     return ComparisonLevel(level_dict, sql_dialect=_mutable_params["dialect"])
+
+
+def distance_in_km_level(
+    km_threshold: Union[int, float],
+    lat_lng_array: str = None,
+    lat_col: str = None,
+    long_col: str = None,
+    not_null: bool = False,
+    m_probability=None,
+) -> ComparisonLevel:
+    """Use the haversine formula to transform comparisons of lat,lngs
+    into distances measured in kilometers
+
+    Arguments:
+        km_threshold (int): The total distance in kilometers to evaluate your
+            comparisons against
+        lat_lng_array (str): The column name for a concatenated array containing both
```

Comment:
I think this is typically called a `struct` (see spark and duckdb) rather than an `array`